src/EnergyPlus/DataDaylighting.hh

```diff
@@ -288,11 +280,6 @@ namespace DataDaylighting {
         Array4D<Real64> DaylSourceFacSunDisk;
         Array4D<Real64> DaylBackFacSun;
         Array4D<Real64> DaylBackFacSunDisk;
-
-        // Default Constructor
-        MapCalcData() : TotalMapRefPoints(0), Zone(0)
-        {
-        }
```

Comment:
So there is no need for a constructor/destructor? In my case/branch, is this needed or not?

        UnitarySys() = default;
        ~UnitarySys() = default;